homeassistant/components/gpsd/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "gpsd",
   "name": "GPSD",
   "documentation": "https://www.home-assistant.io/integrations/gpsd",
-  "requirements": ["gps3==0.33.3"],
+  "requirements": ["agps3==0.36.1"],
```

Comment:
How have you tried to contact the maintainer of gps3?

Reply:
I haven't.  Based on the dates of the outstanding pull requests and issues and lack of response form the owner I didn't bother.

Reply:
Please try to contact the maintainer. Maybe they are willing to transfer ownership of the repo or make you a maintainer?

We want to avoid contributing to spawning forks of projects. That should be the last resort.